src/commands/lambda/flare.ts

```diff
@@ -30,7 +30,9 @@ const FLARE_OUTPUT_DIRECTORY = '.datadog-ci'
 const LOGS_DIRECTORY = 'logs'
 const FUNCTION_CONFIG_FILE_NAME = 'function_config.json'
 const ZIP_FILE_NAME = 'lambda-flare-output.zip'
-const LOG_STREAM_COUNT = 3
+const MAX_LOG_STREAMS = 50
+const DEFAULT_LOG_STREAMS = 3
+const MAX_LOG_EVENTS_PER_STREAM = 1000
```

Comment:
I chose the values for `MAX_LOG_STREAMS` and `MAX_LOG_EVENTS_PER_STREAM` somewhat arbitrarily.

I don't really see a scenario where support needs to see past the last 50 log streams, but let me know if there would be a better value for this const.

Assuming each log event is about 200 characters long, a log stream with 1,000 log events would be about 200KB.

Meaning there could a maximum file size of about 200KB per stream * 50 streams = 10MB, but the real size would likely be far far less.

If 10MB is too large, we can lower either `MAX_LOG_STREAMS` or `MAX_LOG_EVENTS_PER_STREAM`.

Reply:
Thanks for the in-depth explanation and calculations. Sounds about right.